include/mconfig.h

```diff
@@ -84,7 +84,9 @@
 #define MCFG_NONE 0
 #define MCFG_SCALE 1
 #define MCFG_SIZE 2
-#define MCFG_PRESSURE 3
+#define MCFG_PRESSURE_SIZE 3
+#define MCFG_SIZE_PRESSURE 4 /* same capabilities as above, but with higher resolution of touches*/
```

Comment:
This should include renaming the MCFG_PRESSURE variable. They should all be MCFG_PRESSURE_TYPE where the TYPE denotes the specific type of pressure measurement/resolution/whatever that is being used.